tools/online_editor/styles/content.css

```diff
@@ -136,10 +132,106 @@
   width: 100%;
 }
 
+.properties-editor .properties-table .group-header td {
+  background-color: #000000;
+  color: #ffffff;
+  font-weight: bold;
+}
+
 .properties-editor .properties-table .undefined {
   background-color: #b0b0b0;
 }
 
+.properties-editor .value-column.type-unknown:before {
+  content: "??? ";
+  padding: 2px;
+}
+
+.properties-editor .value-column.type-angle:before {
+  content: "ANG ";
+  padding: 2px;
+}
+
+.properties-editor .value-column.type-bool:before {
+  content: "BOO ";
+  padding: 2px;
+}
```

Comment:
To be honnest, i think it would be better to leave that out for now until they have some icons for it.